WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderWebView.java

```diff
@@ -95,10 +95,15 @@ private void init() {
             this.setWebChromeClient(mReaderChromeClient);
             this.setWebViewClient(new ReaderWebViewClient(this));
             this.getSettings().setUserAgentString(WordPress.getUserAgent());
-            this.getSettings().setDefaultFontSize(
-                    (int) (this.getSettings().getDefaultFontSize() * getResources().getConfiguration().fontScale));
-            this.getSettings().setDefaultFixedFontSize(
-                    (int) (this.getSettings().getDefaultFixedFontSize() * getResources().getConfiguration().fontScale));
+
+            if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.KITKAT) {
```

Comment:
Since the problem only occurs when the font scale isn't "normal," how about we further isolate this and only adjust the sizes when `fontScale != 1.0f`? Also, just in case my earlier comment was missed, this could really benefit from a comment explaining its purpose.